### ISSUE TITLE
[A11yMAS][Accessibility][Keyboard][MAS2.4.7][Eclipse][Wipro]:  Keyboard focus is not visible for "Available packs" to install in "Available Software Window"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1347]
**Repro Steps:**

1.  Open and Start Eclipse.
2.   Press "Alt+H" to open Help menu.
3.  Navigate and select Install new software Menu item under Help menu.
4.  In Available Software dialog , navigate to Work with text edit box and provide http://dl.microsoft.com/eclipse/ link and hit enter.
5.  Navigate up to "Type filter text" edit field then press tab and observe the keyboard focus

**Expected Result:**
Keyboard focus should be visible for checkbox's for available packs to install present under Name.

**Actual Result:**
Keyboard focus is not visible for checkbox's for available packs to install present under Name
**Observation:** keyboard focus is not visible for available packs, but by using arrow keys we are able to access the available packs. Once user traverse the available packs we are able to see the focus for the next time navigation.

![keyboard focus is not visible for available packs to install in available software window](https://user-images.githubusercontent.com/22956130/35388068-5117dc04-01f8-11e8-88a0-a6894e05ff92.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix this.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=534670

**Comment by jdneo** [comment]
User can achieve the focus by using the down arrow. See the gif below:
![focus](https://user-images.githubusercontent.com/6193897/43437868-c8159cda-94bd-11e8-8dff-833bc528dc10.gif)

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a) .Issue Still Repros .

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a)
OS Version - 1803 (OS Build 17134.167)
 .Issue Still Repro .
![focus visible](https://user-images.githubusercontent.com/22956130/43827626-eb3ac504-9b17-11e8-8afd-3a601725a98e.gif)

**Comment by jdneo** [comment]
@WiproPSAccTest When the focus is here:
![image](https://user-images.githubusercontent.com/6193897/43828840-2fbafab0-9b30-11e8-81c7-702e05cbf840.png)

Can you try to use down arrow to switch the focus?

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's about the one-time installation.
* **Quality of experience**: not impacted. A simple workaround is available, users can install the plugin from **Help->Marketplace** instead of **Help->Install New Software**.  
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749832
